<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSlider" name="slider" >
     <property name="maximum" >
      <number>5</number>
     </property>
     <property name="value" >
      <number>3</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushbutton" >
     <property name="text" >
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="togglebutton" >
     <property name="text" >
      <string>ToggleButton</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_cameraPos" >
     <property name="text" >
      <string>DEFAULT TEXT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox" >
     <item>
      <property name="text" >
       <string>Paths Of Glory</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Lolita</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Dr. Strangelove</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>2001: A Space Odyssey</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Clockwork Orange</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Barry Lyndon</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>The Shining</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Full Metal Jacket</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Eyes Wide Shut</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit" />
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit" >
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
